brave/brave-core
# Add always show bookmark bar on NTP option

Fix https://github.com/brave/brave-browser/issues/4782

Some user might want to show bookmark on NTP
when show bookmark bar option is false.

Like below, only NTP has bookmark bar when this option is on.
<img width="926" alt="Screen Shot 2019-07-05 at 15 10 14" src="https://user-images.githubusercontent.com/6786187/60701611-4d469600-9f37-11e9-8569-05d3485e0cd1.png">


## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
`yarn test brave_browser_tests --filter=BookmarkTabHelperBrowserTest.AlwaysShowBookmarkBarOnNTPTest`

1. Enable always show bookmark bar on new tab page option
2. Check bookmark bar is visible on new tab

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
